app/Http/Controllers/TaskController.php

```diff
@@ -29,26 +29,44 @@ public function index(Request $request)
         return view('task.index', [
             'tasks' => $tasks,
             'categories' => $categories,
-            'selectedCategory' => $request->category_id
+            'selectedCategory' => $request->category_id,
+            'tags'=>Tag::all(),
         ]);
     }
 
     public function create()
     {
         $categories = Category::all();
-        return view('task.create');
+        $tags = Tag::all();
+        return view('task.create',compact('categories','tags'));
```

Comment:
why are you using compact in create method and a simple array on the method index?